WordPress/src/main/res/layout/my_site_info_card.xml

```diff
@@ -4,7 +4,6 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginBottom="@dimen/margin_medium"
     android:paddingEnd="@dimen/margin_medium_large"
     tools:ignore="RtlSymmetry">
```

Comment:
Suggestion (💡): As I understand you tried to make every card adhere to the same style, in terms of the parent padding. Thus, I want to suggest the below:
- For `my_site_info_card.xml` layout, we can add this on the `root` view:
```
    android:paddingEnd="@dimen/margin_medium_large"
    android:paddingStart="@dimen/content_margin_site_row_start"
    android:paddingTop="@dimen/margin_extra_medium_large"
    android:paddingBottom="@dimen/margin_extra_large"
```
- We can then delete these from the `blavatar_container` view:
```
        android:layout_marginStart="@dimen/content_margin_site_row_start"
        android:layout_marginTop="@dimen/margin_extra_medium_large"
        android:layout_marginBottom="@dimen/margin_extra_medium_large"
```
- And also, this can be deleted as unnecessary as well from the `switch_site` view:
```
        android:layout_marginTop="@dimen/margin_extra_medium_large"
```

Then, for the `my_site_quick_actions_card.xml` layout, we can add the top margin as:
```
    android:layout_marginBottom="@dimen/margin_medium"
    android:layout_marginEnd="@dimen/margin_extra_large"
    android:layout_marginStart="@dimen/margin_extra_large"
    android:layout_marginTop="@dimen/margin_medium"
```

This will make this layout and the rest of the card layouts very similar, in terms of the `root` view layout margin styling. We can then extract that to a style of its own and make sure that every new card will implement this style in its root from now onwards, thus making it easy to preserve consistency of such card layouts. Wdyt?

Reply:
Nice suggestion 👍 
Accepted in 91e553c.

Reply:
👋 @ashiagr !

You are the best!

> We can then extract that to a style of its own and make sure that every new card will implement this style in its root from now onwards, thus making it easy to preserve consistency of such card layouts.

Do you also think that extracting now the below into a style and reusing in every such card layout file can be done as well? I am seeing there already exist a style, the `WordPress.CardView.Unelevated`, maybe we could create a new style, extending from that and add this margins to it, then, every card will just apply this style to get its margins right, kind of enforcing those card layout to be doing that?

```
    android:layout_marginBottom="@dimen/margin_medium"
    android:layout_marginEnd="@dimen/margin_extra_large"
    android:layout_marginStart="@dimen/margin_extra_large"
    android:layout_marginTop="@dimen/margin_medium"
```

Reply:
Nice idea @ParaskP7!

How about we use an item decorator for this purpose which can give us better control of the recycler view items margins?

Reply:
Aw yea, love it @ashiagr ! ❤️

Reply:
Perfecto! 🚀